maps/torch/job/torch_jobs_boh.dm

```diff
@@ -33,14 +33,11 @@
 
 /datum/job/cmo
 	allowed_branches = list(
-		/datum/mil_branch/fleet,
-		/datum/mil_branch/marine_corps = /decl/hierarchy/outfit/job/torch/crew/command/cmo/marine
```

Comment:
I see no reason to remove the CMO from being a marine, military does indeed have own doctors in some countries, actual doctors in Military Hospitals. That goes for the entirety of medbay